src/droplet.F

```diff
@@ -550,64 +554,110 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
 
      allocate(holes_ind(numHoles))
 
-     !NEW !$acc data create(Depart_ind,holes_ind)
-     !NEW !$acc parallel loop gang vector default(present)
+     i=0
+
      do np = 1, numHoles
        holes_ind(np) = undefined_index
      end do
 
-     i=1
-     !NEW !$acc parallel loop gang vector default(present)
      do np = 1, nparcelsLocal
        if ( (pdata_locind(np,1) .eq. undefined_index) .and. &
             (pdata_locind(np,2) .eq. undefined_index) ) then
-          !NEW !$acc atomic capture
-          holes_ind(i) = np
           i = i + 1
-          !NEW !$acc end atomic
+          holes_ind(i) = np
        end if
      end do
 
      if(timestats.ge.1) time_dropC1=time_dropC1+mytime()
 
-#ifdef MPI
      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
      ! Step 2: Setup the indirect addresses for the droplets that are departing !
      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-    call setupIndexPointers(Depart,ptrDepart)
+     ! On the CPU, assign pointers (ptrDepart) into indirect address 
+     ! array (Depart_ind) for deprting droplets
```

Comment:
Should it be `Depart` rather than `Depart_ind`?

Reply:
The variable 'Depart' is the number of particles departing to each neighbor.  The variable 'ptrDepart' is the pointer to the beginning of the indirect address arrays (Depart_ind) that correspond to each direction.

Reply:
Thanks for the clarification.